docs/Documentation/Scripting/Building-Blocks/Integration-List.md

```diff
@@ -530,6 +530,31 @@ The syntax is as follows:
 Check the [Luck Perms documentation](https://luckperms.net/wiki/Context)
 for an in-depth explanation on what contexts are and how to add them to permissions.
 
+### Permissions
+In case you still want to add and remove permissions directly and dont want to have the chat spam when you use the
+command events, you can use the `luckperms addPermission' and `luckperms removePermission` events. It's designed to
+handle the most important functions of LuckPerms.
+
+ ```YAML title="Example"
+ events:
+   addDefaultGroup: "luckperms addPermission permission:group.default,group.quester" #(1)!
+   addNegated: "luckperms addPermission permission:tutorial.done value:false" #(2)!
+   addWithContext: "luckperms addPermission permission:group.legend context:server;lobby" #(3)!
+   addTemporary: "luckperms addPermission permission:donator.level.one expiry:20 unit:MINUTES" #(4)!
+   removeTutorial: "luckperms removePermission permission:tutorial.done"
+   removeMultiple: "luckperms removePermission permission:tutorial.done,group.default" #(5)!
+ ```
+
+1. You can define single or multiple permissions with the `permission` key. You need to separate them with a comma.
+2. You can define Permissions with a optional `value` of `false` to negate them and give them to the player. If you want to override the value of the permission, you can use the `value` argument and set it to `true`.
+3. You can also add optional `context`s to the permissions like `server;lobby`. Read more about contexts [here](https://luckperms.net/wiki/Context). You can define multiple contexts by separating them with a `,`.
```

Comment:
1 and 3 should use consistent wording, so i recommend comma as word, not as `,`
same for 5